Documentation~/Diagnostics.md

```diff
@@ -50,3 +50,5 @@ This is a full list of all builtin settings diagnostics:
 | PAS1000 | Hybrid Rendering Static batching            | Player    | Any                      |
 | PAS1001 | Lit Shader Mode Forward and Deferred        | HDRP      | Any                      |
 | PAS1002 | Camera Lit Shader Mode Forward and Deferred | HDRP      | Any                      |
+| PAS1003 | Texture: Mip Maps not enabled               | Graphics  | Any                      |
```

Comment:
We should probably differentiate Setttings issues from assets specific ones. I am thinking, in this case we could use the PAT prefix (Project Auditor Texture)